mlflow/tracking/fluent.py

```diff
@@ -127,6 +127,37 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
     :param nested: Controls whether run is nested in parent run. ``True`` creates a nest run.
     :return: :py:class:`mlflow.ActiveRun` object that acts as a context manager wrapping
              the run's state.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create nested runs
+        with mlflow.start_run(run_name='PARENT_RUN', nested=True) as parent_run:
```

Comment:
```suggestion
        with mlflow.start_run(run_name='PARENT_RUN') as parent_run:
```

Reply:
Right, you only need the child to be nested. good point!